modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/DeployService.java

```diff
@@ -104,7 +104,7 @@ public class DeployService {
     private ServiceOrderManager serviceOrderManager;
     @Resource
     private ServiceDeploymentStatusChangePolling serviceDeploymentStatusChangePolling;
-    @Resource
+    @Resource(name = "xpanseAsyncTaskExecutor")
```

Comment:
can we use the name from here - https://github.com/swaroopar/osc/blob/191b44b81a1113b5675a64260e2b046aac918c36/modules/async/src/main/java/org/eclipse/xpanse/modules/async/TaskConfiguration.java#L21